GitUI/CommandsDialogs/FormInit.Designer.cs

```diff
@@ -46,12 +46,12 @@ private void InitializeComponent()
             this.MainPanel.Controls.Add(this.Browse);
             this.MainPanel.Controls.Add(this._NO_TRANSLATE_Directory);
             this.MainPanel.Controls.Add(this.label1);
-            this.MainPanel.Size = new System.Drawing.Size(572, 146);
+            this.MainPanel.Size = new System.Drawing.Size(560, 140);
```

Comment:
Should margin/padding be set for the MainPanel?

Reply:
They are set at `GitExtensionsDialog` level, and inherited here.
However in some situations a descendant may override it, e.g. in `FormStatus` - it has a custom layout and overrides `MainPanel.Padding`.